### PR TITLE
reconstruction of Security section; separating concepts and tasks

### PR DESCRIPTION
**Pull request by dockerymick**
https://github.com/open-cluster-management/backlog/issues/459

 - [x] check master.adoc for security section

### CODE REVIEW

File: governance/config_policy_ctrl.md [hunk 1]
```diff
@@ -1,142 +1,111 @@
...
+|Field|Description|
+|-- | -- |
+| apiVersion | Required. Set the value to `policy.mcm.ibm.com/v1alpha1`. <!--current place holder until this info is updated--> |
+| kind | Required. Set the value to `Policy` to indicate the type of policy. |
+| metadata.name | Required. The name for identifying the policy resource. |
+| metadata.namespaces | Optional. |
+| spec.namespace | Required. The namespaces within the hub cluster that the policy is applied to. Enter parameter values for `include`, which are the namespaces you want to apply to the policy to. `exclude` specifies the namespaces you explicitly do not want to apply the policy to.**Note**: A namespace that is specified in the object template of a policy controller, overrides the namespace in the corresponding parent policy.|
+| remediationAction | Optional. Specifies the remediation of your policy. The parameter values are `enforce` and `inform`. **Important**: Some policies may not support the enforce feature.|
+| disabled | Required. Set the value to `true` or `false`. The `disabled` parameter provides the ability to enable and disable your policies.|
+| spec.complianceType | Reqired. Set the value to `"musthave"`|
```
Comment (cadawson) [inline governance/config_policy_ctrl.md:104]: "Required" typo

File: governance/config_policy_ctrl.md [hunk 2]
```diff
@@ -1,142 +1,111 @@
...
-You can view any configuration policy and its status from the console.
-
-1. Log in to your cluster from the console.
-2. From the navigation menu, click **Govern risk** to view a table list of your policies.
-
-   **Note**: You can filter the table list of your policies by selecting the _All policies_ tab or _Cluster violations_ tab.
-3. Select one of your policies.
-
-For more information, see [Configuration policy samples](../governance/policy_samples.md) to view policy samples that can be applied with the configuration policy controller.
+### Configuration policy YAML table
```
Comment (cadawson) [inline governance/config_policy_ctrl.md:93]: Nice use of a table here. ;-)

File: governance/policy_example.md [hunk 3]
```diff
@@ -1,44 +1,113 @@
-# Red Hat Advanced Cluster Management for Kubernetes policy example
+# Red Hat Advanced Cluster Management for Kubernetes security policy overview
 
-Each Red Hat Advanced Cluster Management for Kubernetes policy can have at least one or multiple templates. For more details about the policy elements, see [Red Hat Advanced Cluster Management for Kubernetes policy overview](../governance/policy_overview.md).
+Use the Red Hat Advanced Cluster Management for Kubernetes security policy framework to create custom policy controllers and other policies. Kubernetes CustomResourceDefinition (CRD) instance are used to create policies. or more information about CRDs, see [Extend the Kubernetes API with CustomResourceDefinitions](https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/).
```
Comment (cadawson) [inline governance/policy_example.md:3]: Subject/verb agreement: K8s CustomResourceDefinition (CRD) instance are...  Sugg: Instances are or instance is?

Comment (cadawson) [inline governance/policy_example.md:3]: Typo For more information

File: governance/policy_example.md [hunk 4]
```diff
@@ -1,44 +1,113 @@
...
+
+Each Red Hat Advanced Cluster Management for Kubernetes policy can have at least one or multiple templates. For more details about the policy elements, view the [Policy YAML table](#policy-parameter-table).
```
Comment (cadawson) [inline governance/policy_example.md:5]: "can have at least one or multiple templates" sounds overly complicated. Can we simplify with something like "...can have one or more templates."?

File: governance/policy_sample_intro.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
+# Red Hat Advanced Cluster Management for Kubernetes policy samples
```
Comment (cadawson) [inline governance/policy_sample_intro.md:1]: Is product name required here in the title?

File: governance/policy_sample_intro.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+
+<!-- need more help add more into the intro maybe?-->
+Create and manage policies in Red Hat Advanced Cluster Management for Kubernetes to define rules, processes, and controls on the hub cluster. View the following policy samples to view how specfic policies are applied:
+
+* [Configuration policy controller sample](config_policy_ctrl.md#config_sample)
```
Comment (cadawson) [inline governance/policy_sample_intro.md:6]: Are these in any particular order?  If not, can we put them in alpha order so they are in some order?  If there is a strong reason for the current order (like this is the order that they are in the file), then it is all good.

File: governance/security.md [hunk 7]
```diff
@@ -1,10 +1,8 @@
 # Security
 
-Manage security and role-based access control (RBAC) of Red Hat Advanced Cluster Management for Kubernetes components.
+Manage your security and role-based access control (RBAC) of Red Hat Advanced Cluster Management for Kubernetes components. Govern your cluster with defined policies and processes, to identify and minimize risks. Use policies to define rules and set controls.
```
Comment (cadawson) [inline governance/security.md:3]: Don't need the comma

File: governance/security.md [hunk 8]
```diff
@@ -1,10 +1,8 @@
...
 
-Manage security and role-based access control (RBAC) of Red Hat Advanced Cluster Management for Kubernetes components.
+Manage your security and role-based access control (RBAC) of Red Hat Advanced Cluster Management for Kubernetes components. Govern your cluster with defined policies and processes, to identify and minimize risks. Use policies to define rules and set controls. 
 
-## Authentication
-
-You can onboard workloads to Identity and Access Management (IAM). You must configure authentication service requirements for  Red Hat Advanced Cluster Management for Kubernetes. For more information, see _Understanding authentication_ in [OpenShift Container Platform documentation](https://docs.openshift.com/container-platform/4.3/authentication/understanding-authentication.html).
+**Prerequisite**: You must configure authentication service requirements for  Red Hat Advanced Cluster Management for Kubernetes to onboard workloads to Identity and Access Management (IAM). For more information see, _Understanding authentication_ in [OpenShift Container Platform documentation](https://docs.openshift.com/container-platform/4.3/authentication/understanding-authentication.html).
```
Comment (cadawson) [inline governance/security.md:5]: Extra space before Red Hat.

File: governance/security.md [hunk 9]
```diff
@@ -1,10 +1,8 @@
...
 
 ## Role-based access control (RBAC)
```
Comment (cadawson) [inline governance/security.md:7]: Remove RBAC acronym from title. It is defined in the text.

File: governance/security.md [hunk 10]
```diff
@@ -35,8 +33,7 @@ View the RBAC table for the Red Hat Advanced Cluster Management console:
 | Provider conections | create, read, update, delete | create, read, update, delete | create, read, update, delete | -|
 {: caption="Table 2. Role-based access control for components in Red Hat Advanced Cluster Management" caption-side="top"}
 
-View the following sections for information about managing your cluster security:
+Review the following topics to learn more about securing your cluster:
```
Comment (cadawson) [inline governance/security.md:36]: Good change!

File: cert_manager/certificates.md [hunk 11]
```diff
@@ -2,32 +2,24 @@
 
 Various certificates are created and used throughout Red Hat Advanced Cluster Management for Kubernetes.
 
-After you create your certificates, you can replace certain certificates that are created by the Red Hat Advanced Cluster Management for Kubernetes installer. You must be a cluster or team administrator. See the following documents to learn more information about managing certificates created by the installer:
+After you create your certificates, you can replace certain certificates that are created by the Red Hat Advanced Cluster Management for Kubernetes installer. You must be a cluster administrator or team administrator. See the following documents to learn more information about managing certificates created by the installer:
 
 * [Replacing the root CA](cert_root_ca.md)
 * [Replacing the management ingress certificates](cert_mgmt_ingress.md)
```
[the remark this answers is not in the document]
Comment (dockerymick) [reply cert_manager/certificates.md:6]: I believe users are only able to replace the certs on the hub cluster

File: cert_manager/certificates.md [hunk 12]
```diff
@@ -2,32 +2,24 @@
...
 
   **Note**: Replacing certificates is supported only on native Red Hat Advanced Cluster Management for Kubernetes installations.
 
-## Creating a certificate
```
[the remark this answers is not in the document]
Comment (dockerymick) [reply cert_manager/certificates.md:12]: It should be removed. I point the users to the task (replacing certs) before the note.

File: governance/policy_sample_intro.md (hunk 6 above)
Comment (dockerymick) [reply governance/policy_sample_intro.md:6]: They are in a particular order. The policy samples Memory usage, namespace, image vulnerability policy...SCC policy are Configuration policies. I was attempting to group the samples according to it controller policy. Also, configuration is what we want users to go to first

**Comment by dockerymick** [comment]
Thanks Chris! I have made more updates

**Comment by dockerymick** [comment]
addressed comments. Continuing to merge so that i can create my next PR

**Pull request merged**